src/__tests__/rocketsSlice.test.jsx

```diff
@@ -0,0 +1,62 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] @team its not good practice disabling linters check in your code, kindly try and resolve the linter error instead of disabling the check. Linter checks help us write clean and quality code.